isis/src/qisis/objs/ControlNetTool/ControlNetTool.cpp

```diff
@@ -284,10 +288,23 @@ namespace Isis {
     // TODO: Should we add the SN to the viewPort
     QString serialNumber = SerialNumber::Compose(*vp->cube(), true);
 
-//    if (serialNumber == m_groundSN) {
-//      drawGroundMeasures(vp, painter);
-//      return;
-//    }
+    // Get list of shapes in the ipce project and see if the serial number for viewport passed in
+    // matches any of the project shapes.  If there's a match, draw any ground measures.
```

Comment:
are you drawing ground "measures" or ground points?

Reply:
Changed to ground points.